errai-common/src/main/java/org/jboss/errai/common/metadata/MetaDataScanner.java

```diff
@@ -142,7 +142,7 @@ private MetaDataScanner(final List<URL> urls, final File cacheFile) {
               final Class<?> aClass = Thread.currentThread().getContextClassLoader().loadClass(clsName);
               extensions.add(aClass.asSubclass(Vfs.UrlType.class));
             } catch (final Throwable t) {
-              log.warn("could not load class scanning extension: " + clsName, t);
+              throw new RuntimeException("could not load class scanning extension: " + clsName, t);
```

Comment:
This change just caught my eye as it seems mostly unrelated. What was the motivation here?

Reply:
This is a really good catch! I can't remember when or why I changed that. I'll revert it :)